languages.yaml

```diff
@@ -206,14 +206,14 @@ hu:
 
 de:
   PLUGIN_LOGIN:
-    USERNAME: Nutzername
+    USERNAME: Benutzername
     PASSWORD: Passwort
     ACCESS_DENIED: Zugang verweigert
     LOGIN_FAILED: Login fehlgeschlagen...
     LOGIN_SUCCESSFUL: Du wurdest erfolgreich eingeloggt.
-    BTN_LOGIN: Einloggen
+    BTN_LOGIN: Anmelden
     BTN_LOGOUT: Ausloggen
-    BTN_FORGOT: Vergessen
+    BTN_FORGOT: Passwort vergessen
```

Comment:
I don't think this needs to be `Passwort vergessen` as non of the other languages state what's been forgot. In english it's `forgot` and not `Forgot Password`